virtcontainers/experimental/experimental.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2019 Huawei Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package experimental
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
+	nameRegStr = "^[a-z][a-z0-9_]*$"
+)
+
+// Feature to be experimental
+type Feature struct {
+	Name        string
+	Description string
+	// the expected release version to move out from experimental
+	ExpRelease string
```

Comment:
Nit: I wonder if it's worth creating a `type SemVer string` here? But also, we could validate the string with a package like https://github.com/blang/semver.